themes/custom/lex/scss/_accordions.scss

```diff
@@ -0,0 +1,33 @@
+.ui-accordion {
+    border-radius: 0;
+}
+
+.ui-accordion-header-collapsed, .ui-state-default {
+    background-color: color(med-blue);
+    color: color(white);
+    font-size: 1.4rem;
+    font-family: $font__open_sans;
+}
+
+.ui-accordion-header-active {
+    background-color: color(lex-blue);
+    color: color(yellow);
+    font-size: 1.4rem;
+    font-family: $font__open_sans;
+}
+
+.ui-accordion-content-active {
+    color: black;
+    background-color: color(white);
+    font-family: $font__roboto;
+    font-size: 1rem;
+
+    .accordion-content-heading {
+        font-family: $font__open_sans;
+        font-size: 1.4rem;
+    }
+}
```

Comment:
Starting to get the niceness of using the variables and stuff I see! Nice job incorporating that.